CODEOWNERS

```diff
@@ -14,9 +14,12 @@ azure-pipelines.yml @garyverhaegen-da
 azure-cron.yml      @garyverhaegen-da
 
 # Language
-/compiler/          @hurryabit
-/libs-haskell/      @hurryabit
-/daml-assistant/    @associahedron
+/compiler/          @hurryabit @cocreature
+/libs-haskell/      @hurryabit @cocreature
+/daml-assistant/    @associahedron @hurryabit
```

Comment:
If I do have to be a codeowner I probably should be a codeowner here as well given that I’ve written significant parts of `daml-helper` and the integration tests.